Kickstarter-iOS/DataSources/CommentsDataSource.swift

```diff
@@ -14,8 +14,8 @@ internal final class CommentsDataSource: ValueCellDataSource {
              inSection: Section.comments.rawValue)
   }
 
-  internal func load(project: Project, update: Update?) {
-    self.set(values: [(project, update)],
+  internal func load(project: Project, update: Update?, visible: Bool) {
```

Comment:
seems like we could combine the functionality of this method with the other `load` method. essentially, if we hit `load(comments:project:loggedInUser)` with an empty array of comments then we use the empty state, and otherwise use the comments cell.

we would have to make that method also take an update, but i think overall it would be simpler.